src/components/GameInfoV2/index.tsx

```diff
@@ -0,0 +1,231 @@
+  className?: string
+}> = ({ blockchain, className }) => (
+  <div className={classNames(styles.blockchainIcon, className)}>
+    {React.createElement(blockchain.icon, {
+      className: styles.blockchainSvg
+    })}
+  </div>
+)
+
+const MAX_VISIBLE_BLOCKCHAINS = 5
+
+const GameInfoV2: React.FC<GameInfoV2Props> = ({
+  title,
+  version,
+  earlyAccess,
+  playerCount,
+  image,
+  blockchains = [],
+  showRemainingCount = true,
+  remainingCount = 9,
+  socialLinks,
+  editorChoice,
+  actionButton,
+  i18n,
+  className
+}): JSX.Element => {
+  const renderEditorChoice = () => {
+    if (!editorChoice?.isEditorChoice) return null
+
+    return (
+      <div className={classNames(styles.editorChoice, editorChoice.className)}>
+        <Images.EditorChoice />
+        Editor&apos;s Choice {editorChoice.year || new Date().getFullYear()}
+      </div>
+    )
+  }
+
+  const renderBlockchains = () => {
+    if (!blockchains || blockchains.length === 0) return null
+
+    const visibleBlockchains = blockchains.slice(0, MAX_VISIBLE_BLOCKCHAINS)
+
+    return (
+      <div className={styles.blockchains}>
+        <span className={styles.blockchainLabel}>
+          {i18n?.blockchainLabel || 'BLOCKCHAIN(S):'}
+        </span>
+        <div className={styles.blockchainIcons}>
+          {visibleBlockchains.map((blockchain, index) => (
+            <BlockchainIcon key={index} blockchain={blockchain} />
+          ))}
+          {showRemainingCount && (
+            <div className={styles.more}>
+              <span className={styles.moreCount}>+{remainingCount}</span>
+            </div>
+          )}
+        </div>
+      </div>
+    )
+  }
+
+  return (
+    <div className={classNames(styles.container, className)}>
+      <div className={styles.imageContainer}>
+        {image ? (
+          <GameImage
+            src={image?.src}
+            alt={image?.alt}
+            fallback={image?.fallback}
+            className={styles.image}
+          />
+        ) : null}
+      </div>
+
+      <div className={styles.body}>
+        <div className={styles.content}>
+          <div className={styles.header}>
+            {renderEditorChoice()}
+            <span className={classNames(styles.title)}>{title}</span>
+          </div>
+
+          <div className={styles.info}>
+            <div className={styles.badges}>
+              {version ? (
+                <Sticker styleType="neutral" variant="outlined">
+                  Version {version}
+                </Sticker>
+              ) : null}
+              {earlyAccess ? (
+                <Sticker styleType="neutral" variant="outlined">
+                  Early Access
+                </Sticker>
+              ) : null}
+              <Sticker styleType="neutral" variant="outlined">
+                {title}
+              </Sticker>
+              {playerCount ? (
+                <Sticker styleType="neutral" variant="outlined">
+                  {playerCount}
+                </Sticker>
```

Comment:
left/right padding is off here 8px on the preview and 12px on the design.

![image](https://github.com/user-attachments/assets/c814e7f0-6f95-426f-b548-3bcc694660b4)